kubernetes/samples/scripts/domain-lifecycle/helper.sh

```diff
@@ -0,0 +1,606 @@
+# !/bin/sh
+# Copyright (c) 2020, Oracle Corporation and/or its affiliates.
+# Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+#
+
+#
+# Function to get server start policy at cluster level
+# $1 - Domain resource in json format
+# $2 - Name of cluster
+# $3 - Return value containing cluster level server start policy
```

Comment:
For all enumerated type inputs/outputs on all functions please document the legal values (in this case it can return "NEVER" or "IF_NEEDED" or "".